doc/source/api/solver/solution_variables.rst

```diff
@@ -0,0 +1,158 @@
+  >>> solver.file.read(file_type="case", file_name=import_filename)
+
+
+The solution_variable_info and solution_variable_data objects are attributes of the solver object:
+
+.. code-block:: python
+
+  >>> solution_variable_info = solver.solution_variable_info
+  >>> solution_variable_data = solver.solution_variable_data
+
+
+Solution variable info
+----------------------
+Solution variable metadata information can be accessed via the following solution_variable_info methods:
+
+- ``get_zones_info`` for zone information.
+- ``get_solution_variables_info`` for solution variable information.
+
+Get zone information
+~~~~~~~~~~~~~~~~~~~~
+You can access zone information by calling the ``get_zones_info`` method.
+
+.. code-block:: python
+  
+  >>> zones_info = solution_variable_info.get_zones_info()
+  >>> zones_info.domains
+  ['mixture']  
+  >>>
+  >>> zones_info.zones
+  ['fluid', 'wall', 'symmetry', 'pressure-outlet-7', 'velocity-inlet-6', 'velocity-inlet-5', 'default-interior']
+  >>>
+  >>> zone_info = zones_info['wall']
+  >>>
+  >>> zone_info
+  name:wall count: 3630 zone_id:3 zone_type:wall thread_type:Face
+  >>>
+  >>> zone_info.name 
+  'wall'   
+  >>>
+  >>> zone_info.count 
+  3630 
+  >>>
+  >>> zone_info.zone_id 
+  3
+  >>>
+  >>> zone_info.zone_type 
+  'wall'  
+  
+Get solution variable information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can request solution variables information for a given ``domain_name`` and list of ``zone_names``
+by calling the ``get_solution_variables_info`` method.
+
+.. code-block:: python
+
+  >>> wall_fluid_info = solution_variable_info.get_solution_variables_info(zone_names=['wall' , "fluid"], domain_name="mixture")
+  >>>
+  >>> wall_fluid_info.solution_variables
+  ['SV_CENTROID', 'SV_D', 'SV_H', 'SV_K', 'SV_P', 'SV_T', 'SV_U', 'SV_V', 'SV_W']
+  >>>
+  >>> solution_variable_info_centroid = wall_fluid_info['SV_CENTROID']
+  >>>
+  >>> solution_variable_info_centroid
+  name:SV_CENTROID dimension:3 field_type:<class 'numpy.float64'>
+  >>>
+  >>>solution_variable_info_centroid.name 
+  'SV_CENTROID'
+  >>>
+  >>>solution_variable_info_centroid.dimension 
+  >>>3
+  >>>
+  >>>solution_variable_info_centroid.field_type 
+  <class 'numpy.float64'> 
+  
+Solution variable data
+----------------------
+solution variable data can be extracted and modified via the following solution_variable_data object methods:
+
+- ``get_solution_variable_data`` to get solution variable data.
+- ``set_solution_variable_data`` to set solution variable data.
+
+
+Get solution variable data
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can request solution variable data for a given ``domain_name`` and multiple ``zone_names`` by calling
+the ``get_solution_variable_data`` method and passing the particular ``solution_variable_name``.
+
+.. code-block:: python
+  
+    >>> sv_t_wall_fluid= solution_variable_data.get_solution_variable_data(solution_variable_name="SV_T", zone_names=["fluid", "wall"], domain_name="mixture")
```

Comment:
Similar to the info, this is another example of the long method name that repeats the information in the object name. If we choose `get` for info then we would do the same here.

Reply:
@hpohekar 

**FINAL LIST, AS AGREED:**

```
get_variables_info
get_zones_info
```
```
get_data
set_data
```

Reply:
Thanks a lot. @seanpearsonuk

Reply:
Done. Thank you.